packs/spell-effects/spell-effect-draw-ire.json

```diff
@@ -0,0 +1,107 @@
+{
+    "_id": "0WYy3bY7HJ7TDSYD",
+    "img": "icons/skills/targeting/target-glowing-yellow.webp",
+    "name": "Spell Effect: Draw Ire",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Draw Ire]</p>\n<p><strong>Failure</strong> The target takes full damage and the penalty.</p>\n<p><strong>Critical Failure</strong> The target takes double damage, and the status penalty is -2.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Secrets of Magic"
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.failure",
+                        "value": "failure"
+                    },
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.criticalFailure",
+                        "value": "critical-failure"
+                    }
+                ],
+                "flag": "degreeOfSuccess",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.DegreeOfSuccess"
+            },
+            {
+                "key": "RollOption",
+                "option": "spell-effect:{item|flags.pf2e.rulesSelections.degreeOfSuccess}"
+            },
+            {
+                "hideIfDisabled": true,
+                "key": "FlatModifier",
+                "label": "Draw Ire (Critical Failure)",
+                "predicate": [
+                    "spell-effect:critical-failure",
+                    {
+                        "not": "target:drawIre"
+                    }
+                ],
+                "selector": "attack",
+                "type": "status",
+                "value": -2
+            },
```

Comment:
AdjustModifier this instead of a separate modifier. Remove the labels from both

Reply:
Actually, since we should ditch the adjustment we can just make the values of the choice set -1 and -2 and point back to those without a roll option set